src/Components/Components/test/EventCallbackFactoryBinderExtensionsTest.cs

```diff
@@ -493,7 +493,7 @@ public async Task CreateBinder_NullableDateTimeOffset()
             var expectedValue = new DateTime(2018, 3, 4, 1, 2, 3);
 
             // Act
-            await binder.InvokeAsync(new ChangeEventArgs() { Value = expectedValue.ToString(CultureInfo.InvariantCulture), });
+            await binder.InvokeAsync(new ChangeEventArgs() { Value = expectedValue.ToString(CultureInfo.CurrentCulture), });
```

Comment:
This seems to be unrelated, won't `CurrentCulture` cause issues with machines on other locales?

Reply:
The test was wrong before. Binding works with culture-specific values. The parsing was already culture-specific, so it was incorrect to test a culture-invariant input, and was failing on my machine (using months for days, and vice-versa).

The only reason it was passing in CI before was due to the cultural imperialism of `CultureInfo.InvariantCulture` being equivalent to US date formatting.

Reply:
> the cultural imperialism of `CultureInfo.InvariantCulture` being equivalent to US date formatting

lol, fair enough :).

I only brought it up because we ran into issues in the past in this area.